README.md

```diff
@@ -94,9 +94,7 @@ To declare an API method, call `api.declare(options, handler)` with the followin
  * `route` (required) - the URL pattern, with parameters, e.g., `'/object/:id/action/:param'`
  * `params` - patterns for URL parameters (see below)
  * `query` - patterns for query parameters (see below)
- * `scopes` - scopes required for this API endpoint, in disjunctive normal form (see below)
- * `deferAuth` - if true, authentication will not be checked automatically before the handler
-   is invoked.  In this case, the handler must call `req.satisfies()`; see below.
+ * `scopes` - scopes required for this API endpoint, in 'scope expression' form (see below)
```

Comment:
on vidyo we discussed adding a `terms` property that makes from parameters to explanations. This could be done in a follow up..